dev-embedded/openocd/openocd-0.11.0_rc1.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools udev
+
+if [[ ${PV} == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="	https://repo.or.cz/openocd.git"
+else
+	MY_PV="${PV/_/-}"
+	MY_P="${PN}-${MY_PV}"
+	S="${WORKDIR}"/${MY_P}
+	KEYWORDS="~amd64 ~arm ~x86"
+	SRC_URI="mirror://sourceforge/project/${PN}/${PN}/${MY_PV}/${MY_P}.tar.gz"
+fi
+
+DESCRIPTION="OpenOCD - Open On-Chip Debugger"
+HOMEPAGE="http://openocd.sourceforge.net"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="capstone +cmsis-dap dummy +ftdi +jlink parport +usb verbose-io"
+RESTRICT="strip" # includes non-native binaries
+
+RDEPEND="
+	acct-group/plugdev
+	>=dev-lang/jimtcl-0.76:0=
+	capstone? ( dev-libs/capstone )
+	cmsis-dap? ( dev-libs/hidapi )
+	jlink? ( dev-embedded/libjaylink )
+	usb? (
+		virtual/libusb:0
+		virtual/libusb:1
+	)
+	ftdi? ( dev-embedded/libftdi:= )"
+
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-configCapstone.patch
```

Comment:
If we have only one patch, you squash the PATCHES array into a single line
i.e. `PATCHES=( "${FILESDIR}"/${P}-configCapstone.patch )`